app/backend/src/clj/teet/admin/admin_queries.clj

```diff
@@ -0,0 +1,12 @@
+(ns teet.admin.admin-queries
+  (:require [teet.db-api.core :as db-api]
+            [teet.user.user-roles :as user-roles]))
+
+(defmethod db-api/query :admin/list-users [{db :db} _]
+  {:query '[:find (pull ?e [*])
+            :where [?e :user/id _]]
+   :args [db]
+   :result-fn (partial mapv first)})
```

Comment:
Is `(partial mapv first)` something that we want to do basically every time and then possibly something else as well?

Reply:
Not all queries will `pull` maps, but it is a common one for sure.